src/Mono.Android/metadata

```diff
@@ -371,7 +371,7 @@
 	<attr path="/api/package[@name='android.content.res']/class[@name='Resources']/method[@name='getColor']" name="return">Android.Graphics.Color</attr>
 	<attr path="/api/package[@name='android.content.res']/class[@name='TypedArray']/method[@name='getColor']" name="return">Android.Graphics.Color</attr>
 	<attr path="/api/package[@name='android.graphics']/class[@name='Paint']/method[@name='getColor']" name="return">Android.Graphics.Color</attr>
-	<attr path="/api/package[@name='android.graphics']/class[@name='Paint']/method[@name='setColor']/parameter[position()=1]" name="type">Android.Graphics.Color</attr>
+	<attr path="/api/package[@name='android.graphics']/class[@name='Paint']/method[@name='setColor' and parameter[1][@type='int']]/parameter[position()=1]" name="type">Android.Graphics.Color</attr>
```

Comment:
Should we update *every* expression which replaces `int` with `Android.Graphics.Color` to check for `@type`, not just this one?

Reply:
Added.

Note the changes all *should* be no-ops, as there does not currently exist a `long` version.  But this will help ensure that if a `long` version is added we won't break.